src/index.js

```diff
@@ -1,8 +1,12 @@
 import React from 'react';
 import { render } from 'react-dom';
-import App from './app';
-import TestPage from './components/testpage';
+import { BrowserRouter } from 'react-router-dom';
+import 'bootstrap/dist/css/bootstrap.css';
+import Routes from './Routes';
 
-const out = App.prototype ? <App /> : <TestPage />;
-
-render(out, document.getElementById('root'));
+render(
+  <BrowserRouter>
+    <Routes />
+  </BrowserRouter>,
+  document.getElementById('root'),
+);
```

Comment:
I'm wondering why you've removed the App component which is our entry point? could you please wrap **out** constant inside **BrowserRouter**. Again I think it would be more clear if you import **BrowserRout** as **Router**, and by wrapping **out** inside **Router** you will be including **Routes** component as well behind the scene.